src/animations/timeLines.js

```diff
@@ -80,19 +80,27 @@ export const debrisFlyTl = debrisRef => {
   return mainTl;
 };
 
-export const explosionTl = (cloudRef, debrisRef) => {
+export const explosionTl = (cloudRef, debrisRef, isSpinning = false) => {
   const tl = gsap.timeline({
-    defaults: { transformOrigin: '50% 70%', ease: 'slow' },
+    defaults: {
+      transformOrigin: isSpinning ? '50% 50%' : '50% 70%',
+      ease: 'slow',
+    },
   });
 
   return tl
     .addLabel('start')
     .fromTo(
       cloudRef,
-      { autoAlpha: 0, scale: 0 },
+      { autoAlpha: 0, scale: 0, y: isSpinning && '30%' },
```

Comment:
you don't have type checking, so - nobody is complaining, but for sure `y` cannot be `false` - it is a string (or eventualy a number, but not remember atm). you should pass `undefined if is not spinning.